--- libexif-gtk-0.3.5/configure.in~	2007-09-06 16:18:30.000000000 +0200
+++ libexif-gtk-0.3.5/configure.in	2007-09-06 16:23:16.000000000 +0200
@@ -52,7 +52,7 @@
 dnl ---------------------------------------------------------------------------
 CPPFLAGS_save="$CPPFLAGS"
 CPPFLAGS=$LIBEXIF_GTK_CFLAGS
-AC_CHECK_HEADER([exif-mem.h], [
+AC_CHECK_HEADER([libexif/exif-mem.h], [
 	exif_msg=">= 0.6.12"
 	AC_DEFINE(HAVE_EXIF_0_6_12,1,[whether we use a version of libexif greater than 0.6.12])],[
 	exif_msg="< 0.6.12 (think about upgrading)"])
@@ -73,7 +73,7 @@
 
 Configuration (${PACKAGE}):
 
-	Source code location:    $(srcdir)
+	Source code location:    ${srcdir}
 	Version:                 ${VERSION}
 	Compiler:                ${CC}
 
